contrib/scripts/restore_to_newdb.sh

```diff
@@ -0,0 +1,259 @@
+            *) positional_args+=("$1"); shift;;
+        esac
+    done
+}
+
+# validate positional arguments
+function validate_args()
+{
+    if [[ "${#positional_args[@]}" -ne 2 ]]; then
+        show_usage
+    fi
+    asis_dbname="${positional_args[0]}"
+    tobe_dbname="${positional_args[1]}"
+}
+
+# check if tobe_dbname directory exists
+function check_directory_exists() {
+    if [ ! -d "$f_option" ]; then
+        echo "error: directory $f_option does not exist."
+        exit 1
+    else
+        echo "confirmed: directory $f_option exists."
+    fi
+}
+
+# verify backup files exist
+function verify_backup_files() {
+    backup_files=$(ls "$b_option"/"$asis_dbname"_bk* 2>/dev/null)
+    if [ -z "$backup_files" ]; then
+        echo "error: no (${asis_dbname}_bk*) files found."
+        exit 1
+    else
+        echo "confirmed: backup files found."
+        for file in $backup_files; do
+            printf "\tfile: %s\n" "$file"
+        done
+    fi
+}
+
+# copy and modify databases.txt
+function copy_and_modify_databases_txt() {
+    asis_cubrid_databases=$CUBRID_DATABASES
+
+    # Check if databases.txt already exists in $f_option
+    if [ -e "$f_option/databases.txt" ]; then
+        echo "Warning: $f_option/databases.txt already exists."
+        # Optionally, back up the existing file
+        mv "$f_option/databases.txt" "$f_option/databases.txt.bak" || {
+            echo "Failed to back up existing databases.txt."
+            exit 1
+        }
+        echo "Existing databases.txt backed up as databases.txt.bak."
+    fi
+
+    # Copy the original databases.txt to $f_option
+    cp "$asis_cubrid_databases/databases.txt" "$f_option" || {
+        echo "failed to copy databases.txt."
+        exit 1
+    }
+    modify_databases_txt
+
+
+}
+
+# edit databases.txt content
+function modify_databases_txt() {
+    local file_path="$f_option/databases.txt"
+    local temp_file=$(mktemp)
+    local lob_base_path="${f_option}/lob"
+
+    while IFS= read -r line; do
+        if echo "$line" | grep -q "$asis_dbname"; then
+            read -a fields <<< "$line"
+            fields[1]=$f_option
+            fields[3]=$f_option
+            fields[4]="file:$lob_base_path"
+
+            echo "${fields[@]}" >> "$temp_file"
+        else
+            echo "$line" >> "$temp_file"
+        fi
+    done < "$file_path"
+    mv "$temp_file" "$file_path"
+}
+
+# update cubrid_databases environment variable
+function update_cubrid_databases() {
+    export CUBRID_DATABASES=$f_option || {
+        echo "failed to update cubrid_databases."
+        exit 1
+    }
+}
+
+# run restore database command
+function run_restore_db() {
+    local cmd="cubrid restoredb -B $b_option"
+
+    [[ -n "$up_to_date" ]] && cmd+=" -d $up_to_date"
+    cmd+=" -l $level"
+    $partial_recovery && cmd+=" -p"
+    [[ -n "$keys_file_path" ]] && cmd+=" -k $keys_file_path"
+    cmd+=" -u -o db_$(date '+%y%m%d%h%m%s').log $asis_dbname"
```

Comment:
이 부분이 연-월-일-시-분-초 문자열의 생성이 맞다면 아래와 같이 해야할듯 한데요.
'**%h**'는 시간이 아닌듯 합니다.
```suggestion
    cmd+=" -u -o db_$(date '+%y%m%d%H%M%S').log $asis_dbname"
```

Reply:
#[fce209b](https://github.com/CUBRID/cubrid/pull/5591/commits/fce209b12d1daf3b9d256c1e65211cfd81e4ab9b) 에서 수정하였습니다.